### DISCUSSION TITLE
School does not allow signing in without wifi screen (Skiovox)

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1044] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I am on version 120.0.6099.235 trying to perform skiovox. The way that I am doing it is by turning off wifi, entering the kiosk app Questar NY, pressing alt+shift+s ASAP, pressing accessibility, and then the question mark. then from there, I try to press the button to sign in as an existing user and I put in my email and it says that my school has blocked signing in without wifi. Is there any way to easily bypass this screen?

**Comment by Blobby-Boi** [comment, marked as answer]
That happened to me too, but only Sometimes. My solution was to just signin with wifi on and then quickly reload+poweroff and then skiovox.

**Reply by ghost** [reply]
Does it work everytime? and also how fast do you have to reload+power?

**Reply by Blobby-Boi** [reply]
not every time, but most of the time. You have to reload and power in less than 1 second after you log in.
